### PR TITLE
Fix extends

### PR DESCRIPTION
**Pull request by armandfardeau**
#### :tophat: Description
Extends was not properly loaded therefore the first login authorization could not be skipped

### CODE REVIEW

File: lib/extends/controllers/decidim/devise/sessions_controller_extends.rb [hunk 1]
```diff
@@ -1,24 +1,28 @@
...
-  end
 
-  private
+    private
 
-  # Skip authorization handler by default
-  def skip_authorization_handler?
-    ENV["SKIP_FIRST_LOGIN_AUTHORIZATION"] ? ActiveRecord::Type::Boolean.new.cast(ENV["SKIP_FIRST_LOGIN_AUTHORIZATION"]) : true
+    # Skip authorization handler by default
+    def skip_fist_login_authorization?
```
Comment (eliegaboriau) [inline lib/extends/controllers/decidim/devise/sessions_controller_extends.rb:20]: Typo here

**Review by eliegaboriau — APPROVED** [review]
LGTM except for the typo

File: lib/extends/controllers/decidim/devise/sessions_controller_extends.rb (hunk 1 above)
Comment (armandfardeau) [reply lib/extends/controllers/decidim/devise/sessions_controller_extends.rb:20]: Could you explain what is a typo?

Comment (eliegaboriau) [reply lib/extends/controllers/decidim/devise/sessions_controller_extends.rb:20]: You wrote "fist" instead of "first"

Comment (armandfardeau) [reply lib/extends/controllers/decidim/devise/sessions_controller_extends.rb:20]: Thanks you, for this kind of issue could you add a suggestion next time? 
Capture d’écran 2022-11-25 à 18 34 30

Comment (armandfardeau) [inline lib/extends/controllers/decidim/devise/sessions_controller_extends.rb:20]:
```suggestion
    def skip_first_login_authorization?
```

File: lib/extends/controllers/decidim/devise/sessions_controller_extends.rb [hunk 2]
```diff
@@ -1,24 +1,28 @@
...
-      decidim_verifications.first_login_authorizations_path
-    else
-      super
+  extend ActiveSupport::Concern
+
+  included do
+    def after_sign_in_path_for(user)
+      if user.present? && user.blocked?
+        check_user_block_status(user)
+      elsif !skip_fist_login_authorization? && (first_login_and_not_authorized?(user) && !user.admin? && !pending_redirect?(user))
```
Comment (armandfardeau) [inline lib/extends/controllers/decidim/devise/sessions_controller_extends.rb:10]:
```suggestion
      elsif !skip_first_login_authorization? && (first_login_and_not_authorized?(user) && !user.admin? && !pending_redirect?(user))
```

**Comment by armandfardeau** [comment]
@luciegrau What do you think about it?

**Review by luciegrau — APPROVED** [review]

**Review by paulinebessoles — APPROVED** [review]

**Pull request merged**